packages/clarity-js/src/data/metadata.ts

```diff
@@ -209,10 +224,10 @@ function setCookie(key: string, value: string, time: number): void {
           if (i < hostname.length - 1) { 
             // Write the cookie on the current computed top level domain
             document.cookie = `${cookie}${Constant.Semicolon}${Constant.Domain}${rootDomain}`;
-            // Once written, check if the cookie was set successfully
+            // Once written, check if the cookie was set successfully and the value matches exactly with what we set
             // If yes, no more action is required and we can return from the function since rootDomain cookie is already set
```

Comment:
Add more to the comment to explain the problem this important change (checking the value) is fixing with an example. In particular when can the query be set but not return the same value?

Reply:
Sure will do